universe/config.json

```diff
@@ -7,7 +7,7 @@
             "properties": {
                 "name": {
                     "default": "spark",
-                    "description": "The Spark Dispatcher will register with Mesos with this as a framework name.  This service will be available at http://<dcos_url>/service/<name>/",
+                    "description": "The Spark Dispatcher will register with Mesos with this as a framework name.  This service will be available at {{https-protocol}}<dcos-url>/service/<name>/",
```

Comment:
Nit: I know this is probably taken directly from the other change, but I don't see what value this templated value adds here.

Reply:
/shrugs... it's in master though, we should change it there rather than here?

Reply:
This was done to avoid errors with the package linter. It detects misplaced http:// URLs in packaging